File: MathsThing_Plugins/src/uk/ac/uea/mathsthing/plugins/constants/Tau.java
package uk.ac.uea.mathsthing.plugins.constants;

import java.math.BigDecimal;

import uk.ac.uea.mathsthing.IConstantPlugin;

public class Tau extends IConstantPlugin {

	@Override
	public String getName() 
	{
		return "tau";
	}

	@Override
	public BigDecimal getValue() 
	{
		return new BigDecimal(Math.PI * 2.0);
	}

}

File: MathsThing_Plugins/src/uk/ac/uea/mathsthing/plugins/constants/Tau.scala
package uk.ac.uea.mathsthing.plugins.constants

import java.math.BigDecimal

import uk.ac.uea.mathsthing.IPlugin.IConstantPlugin

class Tau extends IConstantPlugin {

  def getName : String = "tau"
    
  def getValue : BigDecimal = new BigDecimal(Math.PI*2)
}